README

```diff
@@ -25,24 +25,33 @@ For more information and details on how to contribute to `htop` visit [htop.dev]
 
 ## Build instructions
 
+### prerequisite
 This program is distributed as a standard GNU autotools-based package.
 
 Compiling `htop` requires the header files for `ncurses` (libncursesw*-dev). Install these and other required packages for C development from your package manager.
 
-Then, when compiling from a [release tarball](https://bintray.com/htop/source/htop), run:
+**Debian/ubuntu**
```

Comment:
If you want to go to the lengths to moronify dependency installation, you probably should do it completely for all supported platforms. A better alternative would be to just mention the required (and optional) libraries required to build. Note that these change for each platform due to different features. `htop` does not only run on Debian/*buntu, but on several other platforms too.

Reply:
I understand what you mean but only listed the required (and optional) libraries. And i think it is imported do to so. 

From a users perspective, that rarely build from source, it is easier to work with some examples what to install. There is 
>   Running `htop` requires `ncurses` libraries (typically named libncursesw*).

*Does that mean i need to install the base library or the  `libncursesw*-dev`?*
*Does all packages managers use the same name for the  `libncursesw*-dev`?*

I currently only run Debian based machines, so it would be wrong of me to state: 
````md

**Redhat/Fedora/CentOS**
~~~ Bash
sudo dnf install libncursesw5-dev autotools-dev autoconf
~~~

````
Without testing it.

Reply:
With the package dependencies there are usually two cases to distinguish: build time and compile time. With `htop` the build time dependencies include all the development headers (on Debian-based distros usually `-dev`), some compiler (`build-essential`) and the actual build system (`autoconf`). If you actually want to run your compiled `htop` binary afterwards, you also need the runtime dependencies, which depend on the features you `configure`.

If you look into the file `configure.ac` you'll notice that especially ncurses is one of those libraries where every system does it's own. In particular: Although `htop` requires ncurses 6.0, the appropriate package is often enough still called `libncurses5` (+`-dev` for headers). Also ncurses usually comes in two flavours: with and without Unicode support. This is also something that is usually reflected in the package name (the additional `w`).

Back to runtime dependencies: `htop` has a set of fixed runtime dependencies, which is kept as minimal as possible. Currently this only includes `ncurses` as well as `libcap` (if capability support is enabled) and `libdl` (i not building static and support for some of the optional libraries is enabled). On most *BSD systems you also have `kvm` as a static requirement to read all the kernel information.

Optional features include things like the `libsensors` readout of temperatures and CPU speeds, or the `systemd` meter, where the build system only needs the appropriate headers at compile time and checks for the availability of the actual runtime lib as `htop` runs.

The full g(l)ory details can be found in `configure.ac` as well in all those files calling `dlopen`.

Which brings us to "What packages do I need?": Usually packages are called similar to the library. Some systems include the `lib` part, others do not. But for all platforms there's a good way to determine the actual name, by looking at the package overview for those systems and looking up, which files they provide.

Reply:
I think it is a great write up, you have done @BenBE. I think it would be great to add a section about dependencies.

For packages theres it to many options like[ Debian/buster](https://packages.debian.org/source/buster/ncurses), 20+ packages, how can a user not familiar with `ncurses`, easily know which to install?

I think 4 lines in a README.md is a well worth it, to bring more users in that can help build from source, for testing.

Reply:
Do you mind adding a proper section regarding the situation with dependencies?